docs/framework/data/adonet/connection-strings.md

```diff
@@ -31,9 +32,11 @@ The quotation marks themselves, as well as the equals sign, do not require escap
 
 Since each value is read till the next semicolon or the end of string, the value in the latter example is `a=b=c`, and the final semicolon is optional.
 
-Although all connection strings share the same syntax described above, the set of recognised keywords depends on the provider and has evolved over the years from earlier APIs such as *ODBC*. The *.NET Framework* data provider for *SQL Server* (`SqlClient`) supports many keywords from older APIs but is generally more flexible and accepts synonyms for many of the common connection-string keywords.
+All connection strings share the same basic syntax described above. The set of recognized keywords depends on the provider, however, and has evolved over the years from earlier APIs such as *ODBC*. The *.NET Framework* data provider for *SQL Server* (`SqlClient`) supports many keywords from older APIs, but is generally more flexible, and accepts synonyms for many of the common connection string keywords.
+
+Spelling mistakes can cause errors. For example, `Integrated Security=true` is valid, but `IntegratedSecurity=true` causes an error.
```

Comment:
This isn't really a spelling mistake; it's a whitespace value in a keyword.

Reply:
@rpetrusha Can you please help me to identify the whitespace value that you're suggesting I fix?

Reply:
What I meant was that `IntegratedSecurity=true` causes an error not because of a spelling mistake (it's spelled correctly), but because a whitespace character is required between "Integrated" and "Security".